.github/workflows/conformance-aks.yaml

```
@@ -143,21 +143,43 @@ jobs:
             az account show
 
       - name: Create AKS cluster
+        id: cluster-creation
         run: |
+          # Create group
           az group create \
             --name ${{ env.name }} \
             --location ${{ env.location }} \
             --tags usage=${{ github.repository_owner }}-${{ github.event.repository.name }} owner=${{ steps.vars.outputs.owner }}
+
+          # Create cluster with a 1 node-count (we will remove this node pool
+          #  afterwards)
+          # Details: Basic load balancers are not supported with multiple node
+          #  pools. Create a cluster with standard load balancer selected to use
+          #  multiple node pools, learn more at aka.ms/aks/nodepools.
```

Let's make this clickable?

```suggestion
          #  multiple node pools, learn more at https://aka.ms/aks/nodepools.
```